stellargraph/data/explorer.py

```diff
@@ -179,22 +188,38 @@ def _check_sizes(self, n_size):
 class UniformRandomWalk(GraphWalk):
     """
     Performs uniform random walks on the given graph
+
+    Args:
+        graph (StellarGraph): Graph to traverse
+        graph_schema (GraphSchema, optional): Graph schema
```

Comment:
I think none of these classes use `graph_schema` at all, so we could potentially not accept the argument in each of them (theoretically we would have `GraphWalk` not call `create_graph_schema` too, which I guess could happen as part of splitting out `RandomWalk`). That is

```suggestion
```

and
```python
def __init__(self, graph, n=None, length=None, seed=None):
    super().__init__(graph, graph_schema=None, seed=seed) # or super().__init__(graph, seed=seed)
```

and similarly for each of the other classes.

Thoughts?

Reply:
Makes sense, I'll update them